about.html

```diff
@@ -0,0 +1,210 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>About WL Global Summit</title>
+    <link
+    rel="stylesheet"
+    href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.9.1/font/bootstrap-icons.css"
+  />
+  <link rel="stylesheet" href="about.css">
+  </head>
+  <body>
+    <!-- header start -->
+    <div
+    class="uk-background-norepeat uk-background-cover uk-background-center-right uk-section"
+  >
+    <div
+      class="uk-position-cover"
+      
+    >
+      <div class="nav-wrapper">
+        <nav class="nav flex-row w-100">
+          <svg
+            id="btn"
+            class="nav_botton hamb hamRotate ham"
+            viewBox="0 0 100 100"
+            width="60"
+          >
+            <path
+              class="line top"
+              d="m 30,33 h 40 c 3.722839,0 7.5,3.126468 7.5,8.578427 0,5.451959 -2.727029,8.421573 -7.5,8.421573 h -20"
+            />
+            <path class="line middle" d="m 30,50 h 40" />
+            <path
+              class="line bottom"
+              d="m 70,67 h -40 c 0,0 -7.5,-0.802118 -7.5,-8.365747 0,-7.563629 7.5,-8.634253 7.5,-8.634253 h 20"
+            />
+          </svg>
+          <div class="logo-wrapper">
+            <a href="/"
+              ><img
+                width="150px"
+                height="auto"
+                src="./image/logo.png"
+                alt="logo"
+            /></a>
+          </div>
+          <div class="menu-container" id="menu-show">
+            <span id="menu-close" class="menu-close"
+              ><nav class="close-icon">
+                <img src="./image/Close-Icon.png" alt="close-icon" /></nav
+            ></span>
+
+            <div class="menu-list">
+              <ul class="nav nav-list-wrapper flex-row">
+                <li class="nav-item">
+                  <a class="menu-item" href="/index.html">About</a>
+                </li>
+                <li class="nav-item">
+                  <a class="menu-item" href="#program">Program</a>
+                </li>
+                <li class="nav-item">
+                  <a class="menu-item" href="#about">join</a>
+                </li>
+                <li class="nav-item">
+                  <a class="menu-item" href="#about">sponsor</a>
+                </li>
+                <li class="nav-item">
+                  <a class="menu-item" href="#logo-section">News</a>
+                </li>
```

Comment:
- [ ] As mentioned by the previous reviewer you should put the link to the home page and the link to the about page in your menu. As you can see your menu has a link only to the home page which means there is no way to go to the about page in the navigation link, and the link redirects to the home page you name it About. Please kindly fix that 🙏 🙏 

Your menu can look like this 👇 👇 
<img width="327" alt="Screen Shot 2023-01-14 at 22 06 55" src="https://user-images.githubusercontent.com/74501523/212492088-3f9f408f-141b-44a4-baab-631c56b8cc2e.png">

Example of code
<img width="511" alt="Screen Shot 2023-01-14 at 22 09 09" src="https://user-images.githubusercontent.com/74501523/212492112-43e8acca-5fc2-4ee6-a8ec-69ece1df8281.png">